Formula/root.rb

```diff
@@ -62,6 +52,8 @@ def install
               "http://lcgpackages",
               "https://lcgpackages"
 
+    puts "std_cmake_args = #{std_cmake_args}"
```

Comment:
If you just want to find the content of this, it's documented here: https://rubydoc.brew.sh/Formula.html#std_cmake_args-instance_method

You can also check the actual `cmake` invocation when there's a build error: https://github.com/Homebrew/homebrew-core/runs/2938247975?check_suite_focus=true#step:8:55

Reply:
Good to know! I've seen the docs, but needed a simple way to dump it so @vgvassilev could debug locally - looks like the second link was exactly what I needed, though.